src/components/team-members-component/TeamMembersComponent.tsx

```diff
@@ -32,17 +47,26 @@ export const TeamMembersComponent = (props: TeamMembersComponentProps) => {
         <NewTeamMember onCreate={createTeamMembers} />
       </Modal>
       <PageContainer>
-        {teamMembers.map(
-          ({ client, firstName, lastName, role, startingDate }: Types.TeamMember) => (
+        <label htmlFor="sort-team-members">Sort team members:</label>
+
+        <select name="team-members" id="sort-team-members" onChange={handleSortTeamMember}>
+          <option value="">Filter by...</option>
```

Comment:
I'm wondering if this is the right implementation, as "Filter by..." is not an option to choose. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select